openshift/release/fetch-pipeline.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+#
+# Detect which version of pipeline should be installed
+# First it tries nightly
+# If that doesn't work it tries previous releases (until the MAX_SHIFT variable)
+# If not it exit 1
+# It can take the argument --only-stable-release to not do nightly but only detect the pipeline version
+
+# set max shift to 0, so that when a version is explicitly specified that version is fetched
+# modify this in future if a workflow based on latest version and recent (shifted) versions is needed
+MAX_SHIFT=1
+NIGHTLY_RELEASE="https://raw.githubusercontent.com/openshift/tektoncd-pipeline/release-next/openshift/release/tektoncd-pipeline-nightly.yaml"
+STABLE_RELEASE_URL='https://raw.githubusercontent.com/openshift/tektoncd-pipeline/release-${version}/openshift/release/tektoncd-pipeline-${version}.yaml'
+
+function get_version {
+    echo get_get_version
+    local shift=${1} # 0 is latest, increase is the version before etc...
+    local version=$(curl -s https://api.github.com/repos/tektoncd/pipeline/releases | python -c "import sys, json;x=json.load(sys.stdin);print(x[${shift}]['tag_name'])")
+    echo version
+    echo  $version
+    echo $(eval echo ${STABLE_RELEASE_URL})
+}
+
+function tryurl {
+    curl -s -o /dev/null -f ${1} || return 1
+}
+
+function geturl() {
+    if [[ ${1} = "release-next" ]];then
+         if tryurl ${NIGHTLY_RELEASE};then
+             echo ${NIGHTLY_RELEASE}
+             return 0
+         fi
+         echo release next logic\n
```

Comment:
you dont need a \n

Reply:
:+1: 
i think i put it there for debugging.

Reply:
i missed it